core/Twig.php

```diff
@@ -37,16 +42,31 @@ function piwik_filter_truncate($string, $size)
     }
 }
 
+/**
+ * @param string $string
+ * @param int $minFractionDigits
+ * @param int $maxFractionDigits
+ * @return float|int|mixed|string
```

Comment:
`mixed` contains the other types.

```suggestion
 * @return mixed
```